feelpp/feel/feeltask/hardware.hpp

```diff
@@ -0,0 +1,260 @@
+namespace Feel
```

Comment:
there should be a copyright mention at the beginning like in the other Feel++ files